### DISCUSSION TITLE
Re3gistry software at the Google Summer of Code 2023

### DISCUSSION THREAD

**Discussion by jescriu** [ec-jrc/re3gistry#251] [Announcements]
The Re3gistry software is proposing its participation in the Google Summer of Code Program this year, as part of the activities to build an active community around this open source software. 

This could be an opportunity for enthusiastic developers interested in demonstrating their skills, while improving this open source tool and become part of its community. 

The proposal is available at: 
https://github.com/ec-jrc/re3gistry/wiki/Google-Summer-of-Code-2023 

Feel free to use this discussion thread to post any ideas and contact with us on this matter.

**Comment by arantzaetxebarria** [comment]
This is just to inform that [OSGeo](https://wiki.osgeo.org/wiki/Google_Summer_of_Code_2023_Ideas) has been accepted once again as a mentor organization for the [Google Summer of Code 2023](https://summerofcode.withgoogle.com/)

As stated above, the [Re3gistry software](https://joinup.ec.europa.eu/collection/are3na/solution/re3gistry/about) participates also this year, through OSGeo, in this online programme.

If you are a potential contributor, now is the time to discuss the [proposed ideas around the Re3gistry](https://github.com/ec-jrc/re3gistry/wiki/Google-Summer-of-Code-2023) to be ready during the [application period](https://developers.google.com/open-source/gsoc/timeline) from **March 20 - April 4**.

Visit the news item on [joinup](https://joinup.ec.europa.eu/collection/are3na/solution/re3gistry/news/save-date-application-contributors-opens-march-20th)

_Please, share this initiative to reach as many interested people as possible!!!_

**Comment by sdhoundiyal** [comment]
Hey Re3gistry team!
I'm Sandeepan, a grad student at the Indian Institute of Technology (IIT) Bombay working on geospatial solutions for planetary remote sensing. I have experience in knowledge discovery and web development (I worked on integrating ZOO-services with QGIS with ZOO-Project/OSGeo last year) and would like to contribute to the Graphic improvements through GSoC.
I will get started on the test and share updates soon.

**Reply by hernlor** [reply]
Great to hear @sdhoundiyal !
We look forward to receiving your test!

**Reply by sdhoundiyal** [reply]
@hernlor thanks! I just finished the installation. I hadn't worked with Solr before so reading about it took a while. I'm working on the next steps and will hopefully be done soon.

**Reply by sdhoundiyal** [reply]
Hi everyone. I won’t be able to work on the project this summer. I’d had a couple of setbacks with my research, so I will have to devote my time to it. I’m sorry it.

**Reply by juanpelegrina** [reply]
@sdhoundiyal I am sad to hear that, thanks for your interest.

**Comment by aarushi611** [comment]
Hello! I am Aarushi, a third-year Computer Science undergraduate student from Manipal University, India. I am interested in applying to Re3gistry and would love to contribute to your 'Graphic Improvements' project.

I have experience in web development and this project is an excellent opportunity for me to apply my skills and contribute to a real-world project. I have started working on the test of skills and will keep you updated on my progress.

**Reply by juanpelegrina** [reply]
Brilliant @aarushi611, I hope you can collaborate with us. We look forward to receiving your test, let us know if you get stuck at some point.

**Reply by aarushi611** [reply]
Hello. I am writing to update you on my progress with the GSoC application. I am currently working on the test of skills, but I have run into some issues with some of the dependencies required for the setup. I am actively looking into this and hope to resolve these issues as soon as possible.

In the meantime, I have written a rough draft of my project proposal and I would greatly appreciate any feedback or suggestions you may have. Specifically, I am seeking advice on the future developments section of my proposal.

Here is the link to my proposal: https://docs.google.com/document/d/1YRrBUHYp-Z83tBC5F2iKcO3Kt-RiSvysiuaBBaVeUW4/edit?usp=sharing

**Comment by oruscalleda** [comment]
Hi @aarushi611, I've read your proposal and it looks amazing, thank you for your interest!

As for your current issues, let us know any specific problem you're having and we'll try to help you solve it if we can.

Best regards!

**Comment by unaibrrgn** [comment]
Hi @aarushi611 , have you submitted your proposal in your GsoC profile page? Remember deadline April 4th - 18:00 UTC.

Best regards.

**Reply by aarushi611** [reply]
Hello! I have submitted my proposal. I am excited about the opportunity to potentially work with Re3gistry and contribute to the project. Thank you for the help!

**Comment by oruscalleda** [comment]
Hello @aarushi611, I hope you were able to solve those problems you were having in the process of installing the Re3gistry software.
Were you able to complete the proposed test for the Re3gistry GSoC contribution?

Any update on that matter would be appreciated.

Best regards.

**Reply by aarushi611** [reply]
Hello @oruscalleda. I apologise for my delayed response. I had my final exams going on. 
I have set up the database and made all the custumaztions as per the documentation,  however, while installing the Re3gistry 2 core app on tomcat, http://localhost:8080/re3gistry2/install is not working where localhost:8080 is the tomcat server
It is returning error 404. Also, how to do I run the custumization script for tomcat?

**Reply by oruscalleda** [reply]
Hi @aarushi611. Can you give me more details on the installation error? What steps are you able to do? Does the installation page pop-up at all? How have you configured your Tomcat? And the Re3gistry pom file?

About the script, once you modify it to suit your system you can right-click--> execute as administrato (Windows) or navigate through the terminal to the folder where you have the script and type "./scriptname.sh" (Linux). I don't know if this is what you are referring with your question.

**Reply by aarushi611** [reply]
@oruscalleda I have configured all the dependencies and followed all the steps however http://localhost:8080/re3gistry2/install isn't working. And no, the installation page does not pop-up. I'm trying to figure out how to get it to work.
